src/gui/application.cpp

```diff
@@ -758,6 +758,8 @@ void Application::setupTranslations()
                 QLocale newLocale(lang);
                 qCDebug(lcApplication) << "language" << lang << "was enforced, changing default locale to" << newLocale;
                 QLocale::setDefault(newLocale);
+                // setting the layout direction directly only appears to be needed on mac
```

Comment:
IF this doesn't occur on 5.15, then I'd put it in an `if (QT_VERSION < QT_VERSION_CHECK(5, 15, 0))` statement.